src/mscorlib/src/System/Runtime/InteropServices/NonPortable.cs

```diff
@@ -159,37 +159,37 @@ public static bool IsComObject(object o)
 
         public static int QueryInterface(System.IntPtr pUnk, ref System.Guid iid, out System.IntPtr ppv)
         {
-            throw new PlatformNotSupportedException();
+            throw new PlatformNotSupportedException(SR.PlatformNotSupported_ComInterop);
         }
 
         public static int Release(System.IntPtr pUnk)
         {
-            throw new PlatformNotSupportedException();
+            throw new PlatformNotSupportedException(SR.PlatformNotSupported_ComInterop);
         }
 
         public static int ReleaseComObject(object o)
         {
-            throw new PlatformNotSupportedException();
+            throw new PlatformNotSupportedException(SR.PlatformNotSupported_ComInterop);
         }
 
         public static void ZeroFreeBSTR(System.IntPtr s)
         {
-            throw new PlatformNotSupportedException();
+            throw new PlatformNotSupportedException(SR.PlatformNotSupported_ComInterop);
```

Comment:
Why can't ZeroFreeBSTR be supported?  The implementation looks trivial.  Is it just that we don't expect it to be useful?